doc/howtos/booting-and-installation.rst

```diff
@@ -27,19 +27,79 @@ additional build and debugging tools that wouldn't typically be included in a pr
 dev image also will auto-login as ``root`` at the console, something that normally would not be available
 in a production device image but is quite useful during development.
 
-Using dd to Create Bootable Media
-=================================
+
+Using bmaptool to Create Bootable Media
+=======================================
 
 Once you have the :file:`.dsk.xz` Ostro OS image you need to get it
 onto your hardware platform, typically by using removable media such as a
-USB thumb drive or SD card.  The usual way to do this is with the :command:`dd` command.
+USB thumb drive or SD card.
+
+The recommended way to do this is with the :command:`bmaptool` command from `bmap-tools`_.
+A copy of this utility is available in the :file:`deploy/tools` folder after a Yocto Project build
+for your image is finished, or you can install the bmap-tools package with your development system's
+package manager.  For example, on Ubuntu/Debian systems use ::
+
+      $ sudo apt-get install bmap-tools
+
+The ``bmaptool`` program automatically handles copying either compressed and uncompressed images to
+your removable media.  It also also uses a generated ``image.bmap`` file containing a checksum for
+itself and for all mapped data regions in the image file, making it possible to verify data integrity
+of the downloaded image. Be sure to download this ``.bmap`` file along with the image for your device.
+
+
+#. Connect your USB thumb drive or SD card to your Linux-based development system
+   (minimum 4 GB card required, for some images 8 GB card might be required).
+#. If you're not sure about your media device name, use the :command:`dmesg` command to view the system log
+   and see which device the USB thumb drive or SD card was assigned (e.g. :file:`/dev/sdb`)::
+
+      $ dmesg
+
+   or you can use the :command:`lsblk` command to show the block-level devices; a USB drive usually
+   shows up as ``/sdb`` or ``/sdc``
+   (almost never as ``/sda``) and an SD card would usually show up as :file:`/dev/mmcblk0`.
+
+   Note: You should specify the whole device you're writing to with
+   :command:`bmaptool`:  (e.g., :file:`/dev/sdb` or
+   :file:`/dev/mmcblk0`) and **not** just a partition on that device (e.g., :file:`/dev/sdb1` or
+   :file:`/dev/mmcblk0p1`) on that device.
+
+#. The :command:`bmaptool` command will overwrite all content on the device so be careful specifying
+   the correct media device. The ``bmaptool`` opens the removable media exclusively, helping prevent
+   writing to an unintended device.  After verifying your removable media device name, you'll need
```

Comment:
@dbkinder I take the chance to nitpick, mostly out of personal curiosity.
Is it correct to say "helping prevent writing"? To my, admittedly, non-native English-speaker ear, this sounds unnatural.